src/components/mastra/dashboard/ExecutionDashboard.tsx

```
@@ -0,0 +1,287 @@
+                <div>
+                  <span className="font-medium text-green-600">OBSERVE:</span>{' '}
+                  <span className="text-gray-700">{trace.observe}</span>
+                </div>
+              </div>
+            </div>
+          ))}
+        </div>
+
+        <div className="bg-green-50 border border-green-200 rounded-lg p-3">
+          <div className="text-xs font-medium text-green-700">
+            Final Answer: "The top AI agent frameworks in 2025..."
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)